src/components/ecl-breadcrumb/ecl-breadcrumb.tsx

```diff
@@ -17,7 +17,7 @@ export class EclBreadcrumb {
   @Element() el: HTMLElement;
   @Prop() styleClass: string = '';
   @Prop() variant: string = 'default';
```

Comment:
it seems we don't need this prop at all, also because with its default we add a class that has no styles associated.